### PR TITLE
Using new cosmwasm_schema

### PR DESCRIPTION
**Pull request by peara**
I try to implement what's in #81.
Remaining issues:
- In cw2981-royalites, `QueryMsg<Cw2981QueryMsg>` is used. As in cw721-base generates an empty response for `Extension`, there is no response generated for `Cw2981QueryMsg`. I haven't found a solution for this.
- In cw721-non-transferable, most of QueryMsg enums are redefined. I want a more robust solution than copy lines from cw721-base.

### CODE REVIEW

**Comment by peara** [comment]
@shanev @larry0x 
Can you help me check why circle ci didn't run?

**Review by JakeHartnell — DISMISSED** [review]
Looks good to me. CI does appear to be stuck though. @ueco-jb anyway to reset?

File: contracts/cw721-fixed-price/src/msg.rs [hunk 1]
```diff
@@ -22,7 +22,9 @@ pub enum ExecuteMsg {
 }
 
 #[cw_serde]
+#[derive(QueryResponses)]
 pub enum QueryMsg {
+    #[returns(crate::msg::ConfigResponse)]
```
Comment (larry0x) [inline contracts/cw721-fixed-price/src/msg.rs]:
```suggestion
    #[returns(ConfigResponse)]
```

**Review by larry0x — DISMISSED** [review]
LGTM, with one suggested change

**Comment by webmaster128** [comment]
For some strange reason, no CircleCI build is started. I don't know why. I'll set a reminder to check again tomorrow.

**Comment by webmaster128** [comment]
We are getting builds now :)

**Comment by peara** [comment]
It was because I asked Circle ci to check my fork... really sad!

**Comment by webmaster128** [comment]
Strange because fork builds are enabled and they worked for other PRs here before. I see the build in https://app.circleci.com/pipelines/github/CosmWasm/cw-nfts now.

File: contracts/cw721-base/src/msg.rs [hunk 2]
```diff
@@ -125,27 +129,27 @@ pub enum QueryMsg<Q> {
...
-    /// Return type: TokensResponse.
+    #[returns(cw721::TokensResponse)]
     AllTokens {
         start_after: Option<String>,
         limit: Option<u32>,
     },
 
-    // Return the minter
+    /// Return the minter
+    #[returns(crate::msg::MinterResponse)]
```
Comment (larry0x) [inline contracts/cw721-base/src/msg.rs]:
```suggestion
    #[returns(MinterResponse)]
```

**Review by larry0x — DISMISSED** [review]

**Comment by peara** [comment]
> Strange because fork builds are enabled and they worked for other PRs here before. I see the build in https://app.circleci.com/pipelines/github/CosmWasm/cw-nfts now.

Before submitting this PR, I enabled CircleCI on my own fork and that blocked it from running it here.

**Review by larry0x — APPROVED** [review]

**Review by JakeHartnell — APPROVED** [review]

**Pull request merged**